ros_ws/src/simulation/racer_control/include/drive_param_converter.h

```diff
@@ -0,0 +1,97 @@
+#pragma once
+
+#include <ros/ros.h>
+
+#include <drive_msgs/drive_param.h>
+#include <std_msgs/Float64.h>
+
+/**
+ * @brief 
+ * 
+ */
+class DriveParamConverter
+{
+    public:
+    /**
+     * @brief Construct a new Drive Pram Converter object
+     * 
+     */
+    DriveParamConverter();
+    /**
+     * @brief 
+     * 
+     * @param parameters 
+     */
+    void convertDriveParametersCallback(const drive_msgs::drive_param::ConstPtr& parameters);
+    
+
+    private:
+    /**
+     * @brief 
+     * 
+     */
+    struct Angles
+    {
+        double angle1;
+        double angle2;
```

Comment:
Please rename the struct and the doubles to something more descriptive. For example, `AckermannSteeringAngles` and `leftWheelAngle`. That way, you won't need the comments explaining what the variables mean in `drive_param_converter.cpp`, line 59.